exec/java-exec/src/main/resources/rest/query/result.ftl

```diff
@@ -24,6 +24,9 @@
     <link href="/static/css/dataTables.colVis-1.1.0.min.css" rel="stylesheet">
     <link href="/static/css/dataTables.jqueryui.css" rel="stylesheet">
     <link href="/static/css/jquery-ui-1.10.3.min.css" rel="stylesheet">
+    <style>
+      .page-header, .navbar[role=navigation] { display: none }
+    </style>
```

Comment:
you can source a new Shopify style that include this and future changes, so we are only required to modify 1 single line here:
```
<link href="/static/css/shopify-drill.css" rel="stylesheet">
```